external/ilasm/ilasm.depproj

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <!-- Always restore the win7-x64 assets on Windows. These are build-only assets, so they should match the environment of the build. -->
+    <NugetRuntimeIdentifier Condition="'$(RunningOnUnix)' != 'true'">win7-x64</NugetRuntimeIdentifier>
```

Comment:
Sounds good. We essentially want the "runtime RID" which matches the build environment. If we could have something like `$(RuntimeOS)-$(RuntimeArch)`, that would be good.